components/bench_util/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
Seems a very heavyweight choice to add two functions, do you plan to expand this more later?

Reply:
Looks like we have a lot of places better to use a no_cache context to make benchmarks more stable, so I extracted them into a standalone workspace.